.github/workflows/checks.yaml

```diff
@@ -91,6 +91,13 @@ jobs:
     uses: ./.github/workflows/_envtest_tests.yaml
     secrets: inherit
 
+  kongintegration-tests:
```

Comment:
Aren't we adding this to `passed` `needs`?

Reply:
Good catch :) PTAL